shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -517,6 +521,60 @@ - (NSString*)accessibilityLabel {
   return label;
 }
 
+// The point is in global coordinates, so use the global rect here.
+- (bool)containsPoint:(CGPoint)point {
+  return CGRectContainsPoint([self globalRect], point);
+}
+
+- (double)size {
+  const SkRect& rect = [self node].rect;
+  return rect.width() * rect.height();
+}
+
+// Find the smallest eligiable semantics object for _accessibilityHitTest.
+- (SemanticsObject*)search:(CGPoint)point {
+  if ([self children].count == 0) {
+    // Check if the current semantic object should be returned.
+    if ([self containsPoint:point] && [self isFocusable]) {
+      return self;
+    }
+    return nil;
+  }
+
+  SemanticsObject* smallestObject = nil;
+  // Traverse all semantics children to find an eligible and smallest one.
+  for (SemanticsObject* child in [self children]) {
```

Comment:
This children is in traversal order (see accessibility_bridge.mm). We should use hittest order.

Reply:
I dont know what does "use the hittest order" means

Reply:
self.children is derived from https://github.com/flutter/engine/blob/aa0ff2173c6c07cc377b8c8118f006efb6030694/lib/ui/semantics/semantics_node.h#L139 in accessibility_bridge.mm

This for loop need to use the order in childrenInHitTestOrder instead

Reply:
Done